flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "publish-delay-hours": 0
```

Comment:
It doesn't do what you think it does, I'd just drop it. It's not critical to have it up to date immediately.